osf/metrics.py

```diff
@@ -196,6 +197,60 @@ class Index:
             'refresh_interval': '1s',
         }
 
+    class Meta:
+        source = metrics.MetaField(enabled=True)
+
+    @classmethod
+    def filter_institution(cls, institution):
+        return cls.search().filter('match', institution_id=institution._id)
```

Comment:
given the text/keyword thing below, it might be faster/better-practice to filter on the indexed-as-keyword `institution_id.keyword` field, instead of the indexed-as-full-text `institution_id` field

Reply:
disregard, this was my local mappings being out-of-date -- in the current mappings `user_id` is of type `keyword`